name-badges/makebadges.py

```diff
@@ -0,0 +1,40 @@
+#!/bin/pyhton3
```

Comment:
`pyhton3` -> `python3`, though you might want to also change to `#!/usr/bin/env python3` so that this script can be run in a virtualenv easily.